packages/components/index.d.ts

```diff
@@ -254,7 +254,6 @@ interface SelectProps {
   colorFocus?: Color;
   size?: 'medium' | 'large';
   mobileSize?: 'medium' | 'large';
-  placement?: 'top' | 'bottom';
```

Comment:
Shouldn't we preserve but deprecate this prop to keep backward compatibility?

Reply:
I think it's not the "main" prop and we can skip the deprecate option. I will create `alpha` version, instead of main, and we will test changes. Also, we use typescript `3.9.7` (on Hancock for example) and that version doesn't support `@deprecate` field.